src/strings/export.js

```diff
@@ -41,7 +41,18 @@ function stringsToJS(stringsMap) {
   return `export const strings = ${json};\n`;
 }
 
-async function exportStrings(stringsMap, locale, targetDir) {
+async function exportStrings(englishStrings, localizedStrings, locale, targetDir) {
+  const stringsMap = {};
+  for (let key in englishStrings) {
+    if (key in localizedStrings) {
+      stringsMap[key] = localizedStrings[key];
+    } else {
+      // tslint:disable-next-line:no-console
+      console.warn(`Locale ${locale} has no translation for ${key}`);
+      stringsMap[key] = englishStrings[key];
+    }
```

Comment:
should we warn if there are extra keys in the localizedStrings?

Reply:
I think it's harmless if we still have translations for strings we've removed from the English strings file. With this change, they won't be included in the generated JS.

There are questions about deleting strings that I need to address more broadly, but I don't think they'll impact this code. (Basically: how and when strings get completely removed from Phrase. It needs to involve some kind of delay because removing a string from main shouldn't cause us to be unable to do localized patch releases if the prod code still needs the string.)